lib/tests/util/directory.cc

```diff
@@ -1,88 +0,0 @@
-#include <catch.hpp>
```

Comment:
I deleted them because they do basically the same thing as the tests we did write. If you want redundancy/any extra cases they may cover, I'll add them. I didn't find them until after I'd already gone and written my own tests, so my bad.